homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -49,6 +52,12 @@
 
 RESYNC_DELAY = 60
 
+POSKIND_NONE = 0
```

Comment:
Please separate words with underscore in variable names.

Reply:
I'll address this in the next PR to extend this functionality further